src/main/java/io/neonbee/config/NeonBeeConfig.java

```diff
@@ -51,7 +51,7 @@
 
     private String trackingDataHandlingStrategy = DEFAULT_TRACKING_DATA_HANDLING_STRATEGY;
 
-    private List<String> platformClasses = List.of();
+    private List<String> platformClasses = List.of("io.vertx.*", "io.neonbee.*", "org.slf4j.*", "org.apache.olingo.*");
```

Comment:
What exactly is the idea of the prefilled platform classes? Because this will not match ALL platform classes .. Just a best guess? Then I would leave it empty and add these values in the example config, which would be used by the most interested people by default.

Reply:
This is why the default makes more sense than just providing it as an example... If somebody wants to do it differently, it has to be a pretty specific use case.

Reply:
lgtm